vmaas/common_test.go

```diff
@@ -113,34 +113,36 @@ func TestPassReleasever(t *testing.T) {
 //nolint:funlen
 func TestGetRepoIDs(t *testing.T) {
 	updates := Updates{}
+	originalRequest := Request{Organization: "DEFAULT"}
```

Comment:
please test the case which will be used in hosted env when Organization is not defined in the request

Reply:
I will add the coverage, but for requests from vulnerability-engine, we're probably going to add Organization to the request explicitly.

Reply:
Added